portal/server/utils/csv-utils.js

```diff
@@ -76,13 +86,25 @@ const extractCellValue = (cell) => {
     return handleFloatingPointRoundingErrors(cellValue);
   }
 
+  // sets the value to the cellValue - replaced if it is a richText value
+  let value = cellValue;
+
+  /**
+   * if cellValue is a richText value
+   * then needs to be converted to a string
+   * by joining the text values of the richText array
+   */
+  if (isRichTextValue(cellValue)) {
+    value = cellValue.richText.map(({ text }) => text).join('');
```

Comment:
```suggestion
    value = cellValue.richText.filter(({ text }) => text).map(({ text }) => text).join(' ');
```

* Have added filter to ensure no empty is being added.
* Was that suppose to be a space rather than an empty string otherwise exporter name could appear as 'ABCLTD' instead of 'ACB LTD'.

Reply:
this works as it should.  will add filter.  should not be ' ' otherwise it will become A B C L T D

Reply:
This works as should.  Will add the filter.  If ' ', then it would change ACB LTD to A C B  L T D

Reply:
actually ignore me, I will add the ' '